utilities/constants.py

```diff
@@ -89,6 +83,15 @@
         "Swim & Dive",
         "Wrestling",
         "Volleyball"
+    ],
+    "Recreation, Health & Fitness": [
```

Comment:
`Athletics` in line 66 will need to change to Big10 Athletics, right?

Reply:
It looks like I did not think about that change in the issue description. I have updated it now. Thanks.

Reply:
@sandeep-ps looks good now?

Reply:
Yes, thanks.